libs/majority_judgment.py

```diff
@@ -0,0 +1,97 @@
+import math
```

Comment:
didn't take time to look in details. I'll trust the tests :smile:  (see related comment in the tests.py file)

Reply:
Well it came from `moje` which had more tests! We can have some confidence over there!